frontend/components/SuggestPoints.js

```diff
@@ -0,0 +1,350 @@
+import React from 'react';
+import { Button, Modal, Center, Text, VStack, Radio, HStack } from 'native-base';
+import { useState, useEffect } from 'react';
+import { useNavigate, useLocation } from 'react-router-native';
+import FormHeaderLink from './util/FormHeaderLink';
+const data = {
```

Comment:
I think this data should be in separate json file.

Reply:
Probably, but it's here mainly as an example for the backend team, it will be replaced by API call later anyway.